vulture_os/applications/logfwd/models.py

```diff
@@ -214,6 +214,13 @@ def get_rsyslog_template(self):
             # TODO : Distinct ruleset
             for f in Frontend.objects.filter(Q(mode="log") | Q(log_forwarders=self.id)).only('ruleset'):
                 rulesets.add(f.ruleset)
+            for f in Frontend.objects.filter(Q(mode="log") | Q(log_forwarders_parse_failure=self.id)).only('ruleset'):
+                if self.internal:
+                    res += "template(name=\"{}\" type=\"string\" string=\"{}\") \n" \
+                           .format(self.template_id(ruleset=f.ruleset + "_garbage"),
+                                   tpl.render(Context({'ruleset': f.ruleset})).replace("internal", "garbage"))
+            else:
+                rulesets.add(f.ruleset + "_garbage")
```

Comment:
Souldn't this be inside the for loop ?
indentation error ?

Reply:
Yes you were right. It's fixed and tested now.